src/main/com/intellij/lang/jsgraphql/ide/editor/GraphQLIntrospectionHelper.java

```diff
@@ -365,8 +365,10 @@ void createOrUpdateIntrospectionOutputFile(String schemaText, IntrospectionOutpu
 
     void setEditorTextAndFormatLines(String text, FileEditor fileEditor) {
         if (fileEditor instanceof TextEditor) {
+            // IntelliJ only allows the \n linebreak inside editor documents.
+            String normalizedText = text.replaceAll("\\r\\n?", "\n");
```

Comment:
Pls, use `com.intellij.openapi.util.text.StringUtil#convertLineSeparators(java.lang.String)`

Reply:
Hey sorry for the delay, I can definitely update it to this.

Reply:
I've rebased and updated with the `StringUtl` method.